doc/input-output-reference/src/overview/group-condenser-equipment.tex

```diff
@@ -115,19 +115,23 @@ \subsubsection{Inputs}\label{inputs-007}
 
 \paragraph{Field: Design Inlet Air Dry-Bulb Temperature}\label{field-design-inlet-air-drybulb-temperature}
 
-This numeric field specifies the design inlet air dry-bulb temperature of the tower. The default is 35.0\(*^{\circ}\)C.
+This numeric field specifies the design inlet air dry-bulb temperature of the tower. The default is \SI{35.0}{\celsius}.
```

Comment:
This right here 👍 👍 👍 

We need to (eventually) clean up the units throughout all docs with ```\SI```.